includes/features/SU-SWS/stanford_person/stanford_person_cap_node_display.feature

```diff
@@ -3,62 +3,28 @@ Feature: Stanford CAP Person Node Display
   As an end user
   I want to verify the CAP display of people content.
 
-
-  @content @deploy @stanford
-  Scenario: Faculty node
-    Given I am on "people/faculty"
-    And I click "Jacob Smith" in the "Content Body" region
-    Then I should see "Professor of English" in the "Content Body" region
-
-  @content @live @deploy @stanford @safe @contrib @stanford
+  @api @dev @destructive
+  Scenario: Stanford Person
+    Given the "stanford_person" module is enabled
+    And the "entity_token" module is enabled
+    And the cache has been cleared
+    And I am logged in as a user with the "administrator" role
+    And I am on "node/add/stanford-person"
+    When I enter "Leland" for "First name *"
+    And I enter "Stanford" for "Last Name *"
+    And I enter "Leland Stanford" for "Display Name"
+    And I press the "Save" button
+    Then I should be on "people/leland-stanford"
+    And I should see "Person Leland Stanford has been created"
+
+  @api @live
+  Scenario: Affiliates column on Manage Person
+    Given I am logged in as a user with the "administrator" role
+    When I go to "admin/manage/people"
+    Then I should see 1 or more ".views-field-field-s-person-affiliation" elements
+
+  @deploy
   Scenario: Stanford Person CAP node page
```

Comment:
Actually, because the cap node display is different from the default node display, and yet it is part of the feature, we need to separate these tests. They are mutually exclusive.